src/lib/pages/evm-contract-details/components/evm-contract-details-overview/EvmContractCmdGroup.tsx

```
@@ -0,0 +1,85 @@
+import { InteractTabsIndex, TabIndex } from "../../types";
+import { getInteractTabsIndex } from "../../utils";
+
+const getGroupTitle = (interactTab: InteractTabsIndex) => {
+  switch (interactTab) {
+    case InteractTabsIndex.Read:
+      return "Read";
+    case InteractTabsIndex.Write:
+      return "Write";
+    case InteractTabsIndex.ReadProxy:
+      return "Read as Proxy";
+    case InteractTabsIndex.WriteProxy:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)